src/pspm_check_model.m

```diff
@@ -1,63 +1,140 @@
-%   │             data file. See pspm_get_timing for epoch definition;
-%   │             specify a cell array for multiple input files. This
-%   │             must always be specified in SECONDS.
-%   │             Default: no missing values
+%   │
+%   ├───.channel: Definition:
+%   │               * [DCM, SF] channel number
+%   │               * [GLM] channel type
+%   │             Descriptions:
+%   │               * If a channel type is specified the LAST channel
+%   │                 matching the given type will be used.
+%   │               * The rationale for this is that, in general channels
+%   │                 later in the channel list are preprocessed/filtered
+%   │                 versions of raw channels.
+%   │             Special:
+%   │               * If 'pupil' is specified, the function uses the last
+%   │                 pupil channel returned by
+%   │                 <a href="matlab:help pspm_load_data">pspm_load_data</a>.
+%   │                 pspm_load_data loads 'pupil' channels according to a
+%   │                 specific precedence order described in its documentation.
+%   │                 In a nutshell, it prefers preprocessed channels and
+%   │                 channels from the best eye to other pupil channels.
+%   │               * For the modality 'sps', the .channel accepts
+%   │                 only 'sps_l', 'sps_r', or 'sps'.
+%   │             Defaults:
+%   │               * [DCM, SF] 'scr'
+%   │               * [GLM] last channel of the specified modality
+%   ├───.filter:  Definition:
+%   │               * filter settings;
+%   │             Defaults:
+%   │               * modality specific.
+%   ├───.missing: Definition:
+%   │               * Specification of missing epoch files.
+%   │             Descriptions:
+%   │               * Missing epochs should be specified in SECONDs.
+%   │               * If no missing data is sorted, leave undefined.
+%   │               * In default, .missing is not defined, which means
+%   │                 there is no missing data
+%   │             Acceptable values (any of the following):
+%   │               * The name of a .mat file that stores missing epochs
+%   │               * A cell array of multiple .mat files that store
+%   │                 missing epochs
+%   ├─────.norm:  Definition:
+%   │               * Defines whether to normalise data.
+%   │             Acceptable values:
+%   │               * 0: Not to normalise (default)
+%   │               * 1: To normalise
+%   │
+%   │ ▶︎ optional, GLM (modeltype) only
 %   ├───.latency: allows to specify whether latency should be 'fixed'
 %   │             (default) or should be 'free'. In 'free' models an
 %   │             additional dictionary matching algorithm will try to
 %   │             estimate the best latency. Latencies will then be added
 %   │             at the end of the output. In 'free' models the fiel
```

Comment:
Just to confirm both of the two sentences are describing "in 'free' models"? It reads like the last sentence wants to describe another model so it emphasises "in xxx models" at the beginning, but it is repeating "in 'free' models" which makes me feel a little confused.